segmentstore/server/src/main/java/io/pravega/segmentstore/server/attributes/AttributeIndexConfig.java

```diff
@@ -22,9 +22,10 @@
 public class AttributeIndexConfig {
     //region Config Names
 
-    public static final Property<Integer> MAX_INDEX_PAGE_SIZE = Property.named("maxIndexPageSize", 32 * 1024);
-    public static final Property<Integer> ATTRIBUTE_SEGMENT_ROLLING_SIZE = Property.named("attributeSegmentRollingSizeBytes", 4 * 1024 * 1024);
-    private static final int MIN_INDEX_PAGE_SIZE = 1024;
+    public static final Property<Integer> MAX_INDEX_PAGE_SIZE = Property.named("maxIndexPageSizeBytes", 16 * 1024);
+    public static final Property<Integer> ATTRIBUTE_SEGMENT_ROLLING_SIZE = Property.named("attributeSegmentRollingSizeBytes", 16 * 1024 * 1024);
```

Comment:
You mean 1024? I explained this in the `config.props` file. This is the rolling policy for the index segment. Every 32MB (I just changed it), it rolls over the Segment into a new Segment Chunk. This is done so that truncation is effective and that it actually removes obsolete data from the beginning. Too big of a value and we will keep too much data around. Too small of a value and we'll have frequent rollovers, which may be detrimental.